sites/example-project/src/pages/queries/external-queries/+page.md

```diff
@@ -0,0 +1,108 @@
+---
+title: External Queries
+
+sources:
+    - test_query.sql
+    - nested/test_query.sql
+    - dep_query.sql
+    - inline_dep_query.sql
+---
+
+```query_on_the_page
+SELECT 10 as t
+```
+
+Evidence supports using `.sql` files directly; instead of inlining your queries.
+This provides a few benefits:
+ - Reuse common queries across pages
+ - Take advantage of your editor's support for your database (e.g. syntax highlighting, autocomplete, etc.)
+
+<Alert status="warning">
+If you haven't read about <a href="/frontmatter" target="_blank">frontmatter</a> yet, you should start there.
+</Alert>
+
+
+## Using external queries
+
+### Basic Usage
+
+In your project; create a `sources` folder. It should look something like this:
+
+<CodeBlock source={`my_project/
+  pages/
+    my_page.md
+  sources/
+    my_query.sql`}/>
+
+<br/>
+You can now access `my_query.sql` with frontmatter:
+
+<CodeBlock source={`---
```

Comment:
As a bit of an aside, I think you should be able to do:

````
```shell
---    
sources:
  - my_query.sql
---
```
````

and it should render as a codeblock